.github/workflows/linux-v2.yml

```diff
@@ -24,6 +24,7 @@ jobs:
 
     - name: Run Linux Unit Tests
       run: sudo go test -covermode=count -coverprofile=profile.cov -tags azurediskv2 ./pkg/azuredisk --temp-use-driver-v2
+      sudo go test -covermode=count -coverprofile=profile.cov -tags azurediskv2 ./pkg/azdiskschedulerextender
```

Comment:
Using flow scalars works, but might be clearer and more consistent to use literal block scalar here, e.g.
```
      run: |
        sudo go test -covermode=count -coverprofile=profile.cov -tags azurediskv2 ./pkg/azuredisk --temp-use-driver-v2
        sudo go test -covermode=count -coverprofile=profile.cov -tags azurediskv2 ./pkg/azdiskschedulerextender```

Reply:
Fixed